### ISSUE TITLE
Weird scrolling on scaled screen (HDPI)

### ISSUE THREAD

**Issue by OhGhie4iew** [maoschanz/drawing#156]
When running on a high DPI screen where the UI is scaled up, the scrolled window and image size does not correlate. 

In this screenshot, the UI is scaled to 200%. The entire image is filled with red. 

![Drawing - 2020-01-07 13-34-02](https://user-images.githubusercontent.com/7371742/71895864-79470c00-3152-11ea-84c5-a69704029b79.png)

## System

Ubuntu 19.10

**Comment by maoschanz** [comment]
i should have expect it :/ it's not an actual scrolled window it's a joke

**Comment by maoschanz** [comment]
Do the pencil/shapes/lines work correctly at least?

I'm afraid to ask about the selection (and even worse with the version 0.4 you use: the cropping)
